README.md

```diff
@@ -206,6 +208,21 @@ So, could you understand what happened there? Only 3 things on `duix`:
 2. Someone is gonna `subscribe` to a value change, or unsubscribe when component unmount.
 3. Someone is gonna `set` the new value every time it changes
 
+## Advanced options/usages
+
+When subscribing to a value, you can choose special behaviors by passing an object as third parameter:
+
+```lang=javascript
+duix.subscribe(key, callback, {
+  onlyOnChange: false
+});
+```
+
+option        | default value | comment
+--------------|---------------|----------------------------------------------------
+onlyOnChange  | false         | being notified only when the value is different
```

Comment:
What do you think about adding a small doc for callMeNow here?
I can add it if you don’t have free time.

Reply:
I was thinking that "callMeNow" is not a explicit name, I was thinking of "fireImmediately" or something more common... to not mix the discussion, I was planning to open a bug on that subject and make the doc in that bug...

Reply:
Will be fixed somewhere else